code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -65,6 +71,31 @@
 	blood_type = "LE"
 	unique_blood = /datum/reagent/consumable/liquidelectricity
 
+/obj/item/reagent_containers/blood/snail
+	blood_type = "S"
+	unique_blood = /datum/reagent/lube
+	
+/obj/item/reagent_containers/blood/snail/Initialize(mapload, vol)
+	. = ..()
+	desc += " It's a bit slimy... The label indicates that this is meant for snails. "
```

Comment:
Why not just pass this in as something on `examine()`? That would be a bit cleaner and let you have a string without pre-pending a space (as well as have it be on it's very own line in the examine proc) without having to mutate a compile-time variable (which might cause confusion if people want to edit the desc in a future time).

Reply:
fair enough, just easier to not have to deal with the additional args on `Initialize()` when it can be helped really (could just also add the custom `desc` on compile to ensure that it would always make sense, but I know why you did it that way)